.github/workflows/101_build.yml

```diff
@@ -1,14 +1,12 @@
 name: build
 
-permissions: {} 
-
 on:
   workflow_call:
     inputs:
-      skip:
-        description: "Want to skip running certain jobs 'none', 'non-required', 'all'?"
+      jobs_to_run:
+        description: "Which build jobs should be run: 'all', 'only-required', 'skip'?"
```

Comment:
IMO `skip` doesn't make sense the way you renamed. Which jobs should be run? Skip! What?
I'd keep `none`.

Other than that, I'm a fan of configuring the positive instead of negative

Reply:
Obviously the same goes for all the other files

Reply:
changed to `skip-all`

Reply:
I still think that's worse than none, because it's not quite an answer to the question you pose in the description, but won't argue this further